src/fiatExchanges/FiatExchangeCurrency.tsx

```diff
@@ -1,270 +0,0 @@
-import fontStyles from 'src/styles/fonts'
-import variables from 'src/styles/variables'
-import { Network } from 'src/transactions/types'
-import { CiCoCurrency, currencyForAnalytics, resolveCurrency } from 'src/utils/currencies'
-import networkConfig from 'src/web3/networkConfig'
-
-type Props = NativeStackScreenProps<StackParamList, Screens.FiatExchangeCurrency>
-
-export const fiatExchangesOptionsScreenOptions = ({
-  route,
-}: {
-  route: RouteProp<StackParamList, Screens.FiatExchangeCurrency>
-}) => {
-  const { flow } = route.params
-
-  const headerTitle = () => {
-    switch (flow) {
-      case FiatExchangeFlow.CashIn:
-        return i18n.t(`fiatExchangeFlow.cashIn.selectCurrencyTitle`)
-      case FiatExchangeFlow.CashOut:
-        return i18n.t(`fiatExchangeFlow.cashOut.selectCurrencyTitle`)
-      case FiatExchangeFlow.Spend:
-        return i18n.t(`fiatExchangeFlow.spend.selectCurrencyTitle`)
-    }
-  }
-  return {
-    ...emptyHeader,
-    headerLeft: () => (
-      <BackButton eventName={FiatExchangeEvents.cico_currency_back} eventProperties={{ flow }} />
-    ),
-    headerTitle: headerTitle(),
-    headerRightContainerStyle: { paddingRight: 16 },
-  }
-}
-
-// pointerEvents="none" added for disabled testing - https://github.com/wix/Detox/issues/2821
-function CurrencyRadioItem({
-  selected,
-  onSelect,
-  enabled = true,
-  title,
-  body,
-  containerStyle,
-  testID,
-}: {
-  selected: boolean
-  onSelect: () => void
-  enabled?: boolean
-  title: string
-  body?: string
-  containerStyle: ViewStyle
-  testID?: string
-}) {
-  return (
-    <TouchableWithoutFeedback testID={testID} onPress={onSelect} disabled={!enabled}>
-      <View
-        style={[
-          styles.currencyItemContainer,
-          containerStyle,
-          { borderColor: selected ? colors.primary : colors.gray3 },
-        ]}
-      >
-        <RadioButton selected={selected} disabled={!enabled} />
-        <Text style={[styles.currencyItemTitle, enabled ? {} : { color: colors.gray3 }]}>
-          {title}
-        </Text>
-        {body && <Text style={styles.currencyItemBody}>{body}</Text>}
-      </View>
-    </TouchableWithoutFeedback>
-  )
-}
-
-function FiatExchangeCurrency({ route, navigation }: Props) {
-  const { t } = useTranslation()
-  const dispatch = useDispatch()
-  const { flow } = route.params
-
-  const [selectedCurrency, setSelectedCurrency] = useState<CiCoCurrency>(CiCoCurrency.cUSD)
-  // TODO: Update this to actually respect all possible networkIds correctly
-  const showEth = getDynamicConfigParams(
-    DynamicConfigs[StatsigDynamicConfigs.MULTI_CHAIN_FEATURES]
-  ).showCico.includes(networkConfig.networkToNetworkId[Network.Ethereum])
-
-  // Fetch FiatConnect providers silently in the background early in the CICO funnel
-  useEffect(() => {
-    dispatch(fetchFiatConnectProviders())
-  }, [])
-
-  const goToProvider = () => {
-    ValoraAnalytics.track(FiatExchangeEvents.cico_currency_chosen, {
```

Comment:
can we also clean up this analytics event? We unfortunately don't get knip check because it's used in analytics/properties and analytics/docs

Reply:
Fixed c41425ad26304b4429a68183e1d55bffa54c040d & a48ab6a2ae069bf946d45835e2fef6fcf124c78c!